include/deal.II/lac/affine_constraints.templates.h

```diff
@@ -2648,12 +2648,23 @@ AffineConstraints<number>::distribute(VectorType &vec) const
                      "of the locally owned and the locally relevant "
                      "degrees of freedom."));
 
-            // Check that the set of indices we will import is a superset of
```

Comment:
The indentation changes in this patch because clang-format chooses to align comments with the text *above* -- which is here indented by two characters because what's above is guarded by an `if` statement. 

But now this comment is moved below an assertion I add, which is not under the `if` and so un-indented by two characters compared to the previous assertion. As a consequence, the comment is now aligned to this new assertion, and so moves to the left by two characters. The text did not change.